xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/companylogo.vm

```diff
@@ -46,7 +46,11 @@
   #end
 #end
 <div id="companylogo">
-  <a href="$!xwiki.getURL($services.wiki.currentWikiDescriptor.mainPageReference)" title="Home" rel="home" #if(!$displayPageHeader)class="navbar-brand"#end>
-    <img src="$!logourl" alt="Wiki Logo"/>
+  <a href="$!xwiki.getURL($services.wiki.currentWikiDescriptor.mainPageReference)" rel="home" ##
+    title="$escapetool.xml($services.localization.render('core.document.header.companylogo.logo.text'))" ##
+    #if(!$displayPageHeader)class="navbar-brand"#end>##
+    <span class="sr-only">$services.localization.render('core.document.header.companylogo.anchor.text')</span>
+    <img src="$!logourl" ##
+      alt="$escapetool.xml($services.localization.render('core.document.header.companylogo.logo.text'))"/>##
```

Comment:
These translation keys don't match those that are defined in the translation file below.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3510/commits/a5ca06ba999974663502e66327838c1e2b8230b5 :+1: 
Thank you :)